src/kdc/main.c

```diff
@@ -955,6 +955,21 @@ int main(int argc, char **argv)
 
     initialize_kdc5_error_table();
 
+    retval = krb5_get_profile(kcontext, &profile);
```

Comment:
Since this file includes k5-int.h, you could skip this step and just use kcontext->profile.  However, you should instead be reading this in initialize_realms() adjacent to where kdc_max_dgram_reply_size is read.